code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm

```diff
@@ -211,6 +270,17 @@
 				infest(H)
 	..()
 
+/mob/living/simple_animal/hostile/asteroid/hivelordbrood/bloodling/Life()
```

Comment:
All the blood stuff should prpbably be put at the end of this file and not scattered around it, this appears to be halfway through the legion stuff file